src/components/Dashboard/Dashboard.js

```diff
@@ -20,8 +21,71 @@ export const postList = [
     link: `http://localhost:3000/dashboard`,
   },
 ];
+export const kpiList = [
+  {
+    kpiTitle: `Total de Envios`,
+    kpiValue: `21458`,
+    iconClass: `deliveries`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Campaings',
+  },
+  {
+    kpiTitle: `Tasa de Apertura`,
+    kpiValue: `57%`,
+    iconClass: `open-rate`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Campaings',
+  },
+  {
+    kpiTitle: `CTR`,
+    kpiValue: `215%`,
+    iconClass: `ctr`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Campaings',
+  },
+  {
+    kpiTitle: `Total de Contactos`,
+    kpiValue: `21458`,
+    iconClass: `book`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Subscribers',
+  },
+  {
+    kpiTitle: `Contactos Nuevos`,
+    kpiValue: `943`,
+    iconClass: `user-new`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Subscribers',
+  },
+  {
+    kpiTitle: `Contactos removidos`,
+    kpiValue: `94`,
+    iconClass: `user-removed`,
+    kpiPeriod: `ULTIMOS 30 DIAS`,
+    active: true,
+    kpiType: 'Subscribers',
+  },
+];
 
 export const Dashboard = () => {
+  const [kpiListSize, setKpiListSize] = useState({
```

Comment:
`kpiListSize` should not be in the `state` of `Dashboard` 
could be in a normal `constant`